buildSrc/src/main/java/io/micronaut/guides/feature/JakartaPersistenceApi2.java

```diff
@@ -0,0 +1,34 @@
+package io.micronaut.guides.feature;
+
+import io.micronaut.core.annotation.NonNull;
+import io.micronaut.starter.application.ApplicationType;
+import io.micronaut.starter.application.generator.GeneratorContext;
+import io.micronaut.starter.build.dependencies.Dependency;
+import io.micronaut.starter.feature.Feature;
+import jakarta.inject.Singleton;
+
+import static io.micronaut.starter.build.dependencies.Scope.COMPILE;
+
+@Singleton
+public class JakartaPersistenceApi2 implements Feature {
+
+    @NonNull
+    @Override
+    public String getName() {
+        return "jakarta-javax-persistence";
```

Comment:
if we need v 2.2.3 I think a better name would be jakarta-persistence-api-v2

Reply:
I doubt this will be used again, it's only needed for hibernate-jpa which currently uses v2.2.3. I assume it'll be updated to v3.0.0 at some point and we can delete this